diff -up once_cell-1.7.2/Cargo.toml.omv~ once_cell-1.7.2/Cargo.toml
--- once_cell-1.7.2/Cargo.toml.omv~	2021-03-28 18:58:38.197734827 +0200
+++ once_cell-1.7.2/Cargo.toml	2021-03-28 18:59:11.035696486 +0200
@@ -58,7 +58,7 @@ version = "0.11"
 optional = true
 default_features = false
 [dev-dependencies.crossbeam-utils]
-version = "0.7.2"
+version = "0.8"
 
 [dev-dependencies.lazy_static]
 version = "1.0.0"
